vlingo-router-contentbased/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>vlingo-examples</artifactId>
+        <groupId>io.vlingo</groupId>
+        <version>0.7.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>vlingo-router-contentbased</artifactId>
+
+    <properties>
+        <build.name>vlingo-router-contentbased-${version}</build.name>
+        <apache.commons.version>3.7</apache.commons.version>
```

Comment:
Why are `apache.commons` and `google.collect` needed? I don't see the need for these.